### PR TITLE
something test

### PR DESCRIPTION
**Pull request by dombong**
## Description of the task

Please describe the task related to this PR. Or the reason behind the
change.
i edited the desc 

## How to test

How to test the changes? Which command to run? Which output is expected?

```
./monorobot check mock_payloads/SOME_FILE
```

## References

- existing issue:
- Slack discussion:
- other?

### CODE REVIEW

**Comment by dombong** [comment]
new comment

**Comment by dombong** [comment]
new comment i edited this

**Comment by dombong** [comment]
wow what a bot

**Comment by yasunariw** [comment]
comment

File: .monorobot.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+            {
+                "match": [
+                    "documentations/"
+                ],
+                "channel": "general"
+            }
+        ]
+    },
+    "label_rules": {
+        "default_channel": "channel-1",
```
Comment (dombong) [inline .monorobot.json:15]: comment in code

Comment (dombong) [reply .monorobot.json:15]: now?

edit nownwo

Comment (dombong) [reply .monorobot.json:15]: hmmm

**Comment by dombong** [comment]
well then

**Comment by dombong** [comment]
kdkfsdf

File: .monorobot.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    "main_branch_name": "master",
+    "prefix_rules": {
+        "default_channel": "channel-1",
+        "rules": [
+            {
+                "match": [
+                    "documentations/"
+                ],
+                "channel": "general"
+            }
```
Comment (dombong) [inline .monorobot.json:11]: new review

Comment (dombong) [reply .monorobot.json:11]: comment on review

**Comment by dombong** [comment]
new comment

**Comment by dombong** [comment]
heloo there

**Comment by dombong** [comment]
wow

**Comment by dombong** [comment]
im commenting

**Comment by dombong** [comment]
new comment edited

update: edited

File: .monorobot.json (hunk 1 above)
Comment (dombong) [reply .monorobot.json:15]: add in review update review

**Review by dombong — COMMENTED** [review]
ok create

ok edited

File: .monorobot.json [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+    "label_rules": {
+        "default_channel": "channel-1",
+        "ignored_users": [
+            "sewen-thy"
+        ],
+        "rules": [
+            {
+                "match": [
+                    "121"
+                ],
```
Comment (dombong) [inline .monorobot.json:23]: new review

**Review by dombong — COMMENTED** [review]
new review sent

edited new reivew

**Review by dombong — COMMENTED** [review]
imma edit this

nownow

**Pull request closed without merging**